addons/knowledge_website/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': 'Knowledge website',
+    'summary': 'Publish your articles on the web',
+    'version': '0.1',
+    'depends': ['knowledge', 'website'],
+    'data': [
+        'views/knowledge_views.xml'
+    ],
+    'installable': True,
+    'application': False,
+    'auto_install': True,
+    'license': 'LGPL-3',
+    'assets': {
+        'web.assets_backend': [
+            'knowledge_website/static/src/scss/knowledge_views.scss',
+        ]
```

Comment:
missing security files!

Reply:
nope, unless this module changes the security rules.
--> Which is not the case as the access rules are still based on user_has_access and this field is modified to include website_published